.gitlab-ci.yml

```diff
@@ -39,3 +39,24 @@ staging:
         K8S_SECRET_MAILER_EMAIL_BACKEND: "anymail.backends.mailgun.EmailBackend"
         K8S_SECRET_DEFAULT_FROM_EMAIL: "no-reply@hel.ninja"
         K8S_SECRET_FIELD_ENCRYPTION_KEYS: "f164ec6bd6fbc4aef5647abc15199da0f9badcc1d2127bde2087ae0d794a9a0b"
+
+production:
+    variables:
+        K8S_SECRET_ALLOWED_HOSTS: "*"
+        K8S_SECRET_DEBUG: 0
+        K8S_SECRET_OIDC_SECRET: "$GL_HEL_STABLE_TUNNISTAMO_OIDC_SECRET"
+        K8S_SECRET_SECRET_KEY: "$GL_STABLE_DJANGO_SECRET_KEY"
+        K8S_SECRET_SENTRY_DSN: "https://c2a1cc12588942fd938d8cfdea617cb4:6fc6a7eb191a48bf9be329ca344d6dbc@sentry.hel.ninja/33"
+        K8S_SECRET_SENTRY_ENVIRONMENT: "production"
+        K8S_SECRET_SKIP_DATABASE_CHECK: 1
+        K8S_SECRET_TOKEN_AUTH_ACCEPTED_AUDIENCE: "https://api.hel.fi/auth/helsinkiprofile"
+        K8S_SECRET_TOKEN_AUTH_ACCEPTED_SCOPE_PREFIX: "helsinkiprofile"
+        K8S_SECRET_TOKEN_AUTH_AUTHSERVER_URL: "https://api.hel.fi/sso/openid"
+        K8S_SECRET_TOKEN_AUTH_REQUIRE_SCOPE: "True"
+        K8S_SECRET_VERSION: "$CI_COMMIT_SHORT_SHA"
+        K8S_SECRET_MAIL_MAILGUN_KEY: "$SECRET_STABLE_MAILGUN_API_KEY"
+        K8S_SECRET_MAIL_MAILGUN_DOMAIN: "mail.hel.ninja"
+        K8S_SECRET_MAIL_MAILGUN_API: "https://api.eu.mailgun.net/v3"
+        K8S_SECRET_MAILER_EMAIL_BACKEND: "anymail.backends.mailgun.EmailBackend"
+        K8S_SECRET_DEFAULT_FROM_EMAIL: "no-reply@hel.ninja"
+        K8S_SECRET_FIELD_ENCRYPTION_KEYS: "6dda31bbd572d00e6e05c3933b80719770fd6e44645348915c33cf7db1741670"
```

Comment:
Shouldn't this be a secret?

Reply:
Probably yes. I have no access to this stuff in Gitlab.